pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -500,8 +518,11 @@ def append_tk_command_to_qiskit(
     qargs = [qregmap[q.reg_name][q.index[0]] for q in args]
     if optype == OpType.CnX:
         return qcirc.mcx(qargs[:-1], qargs[-1])
-
-    # special case
+    num_controls = len(qargs) - 1
```

Comment:
I think assuming that the "target" gate will always be single-qubit is a bug waiting to happen: better to set this in the individual cases below.

Reply:
In fact I could just do away with the num controls variable and just do 

```
    if optype == OpType.CnY:
        return qcirc.append(qiskit_gates.YGate().control(len(qargs) - 1), qargs)
```

Reply:
Yes, that would be better I think.

Reply:
Okay, done.